wordpress-mobile/WordPress-iOS
# Reader: Map and store `unseen_count` for unread post counts

Ref. #15483 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/319

This PR takes the `unseenCount` value from `RemoteReaderSiteInfo` and stores it in core data as the `unseenCount` attribute in the `ReaderSiteTopic` entity. 

### To test

1. Set a breakpoint in ReaderTopicService.m L55.
2. Check the output in the variables inspector panel. Example: 

`_unseenCount	__NSCFNumber *	(long)3	0xfa871a3e58ac89af`

PR submission checklist:
